bench/archive/jmh-results-20190913120118.json

```diff
@@ -0,0 +1,328 @@
+[
```

Comment:
Should these be a part of the bench repo? How to generate them and visualize?

Reply:
IDK what the bench repo is.